modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/internal/util/MBMessageUtil.java

```diff
@@ -93,7 +257,103 @@ public static MBMessage updateAnswer(
 		return message;
 	}
 
+	private static String _getMarkupElement(
+		MarkupElement element, boolean htmlFormat) {
+
+		if (!htmlFormat) {
+			return StringPool.BLANK;
+		}
+
+		return _markupElements.getOrDefault(element, StringPool.BLANK);
+	}
+
+	private static String _getUserName(
+		boolean htmlFormat, MBMessage mbMessage, String quoteMark) {
+
+		if (!htmlFormat) {
+			return _quotedMessage(false, mbMessage.getUserName(), quoteMark);
+		}
+
+		return mbMessage.getUserName() + "<br />";
+	}
+
+	private static String _quotedMessage(
```

Comment:
Method names should be imperative, they should describe an action. Can you rename this to "quoteMessage" or "getQuotedMessage"?